src/index.js

```diff
@@ -20,4 +21,4 @@ const styled = _styled(_styledComponent(_ComponentStyle(generateAlphabeticName))
 
 /* Export everything */
 export default styled
-export { css, keyframes, injectGlobal, ThemeProvider }
+export { css, keyframes, injectGlobal, ThemeProvider, ServerStyleSheet }
```

Comment:
I think it's more about being consistent 😉 renaming both to `serverStyles` should be fine!